chapter_expressions/reductions.md

```diff
@@ -33,7 +33,7 @@ sum_rows(a, b)
 b
 ```
 
-It's fairly straightforward, we first iterate on the first dimension, `axis=0`, and then sum all elements on the second dimension to write the results. In NumPy, we can use `:` to slice all elements along that dimension.
+It's fairly straightforward, we first iterate on the first dimension, `axis=1`, and then sum all elements on the second dimension to write the results. In NumPy, we can use `:` to slice all elements along that dimension.
```

Comment:
I believe `axis=0` is correct?

Reply:
Uhm, maybe I am confused myself?
I was comparing the description with `numpy` example shown as follow in the text.

```{.python .input  n=29}
a = np.random.normal(size=(3,4)).astype('float32')
a.sum(axis=1)
```

Reply:
I thought about it again, and I think it make sense as a `axis=0` for describing the above function.